controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -132,6 +140,14 @@ func (r *StorageConsumerReconciler) initReconciler(request reconcile.Request) {
 	r.cephClientRBDNode.Name = fmt.Sprintf("%s-%s", "cephclient-rbd-node", r.storageConsumer.Name)
 	r.cephClientRBDNode.Namespace = r.namespace
 
+	r.cephClientCephFSProvisioner = &rookCephv1.CephClient{}
+	r.cephClientCephFSProvisioner.Name = fmt.Sprintf("%s-%s", "cephuser-cephfs-provisioner", r.storageConsumer.Name)
+	r.cephClientCephFSProvisioner.Namespace = r.namespace
+
+	r.cephClientCephFSNode = &rookCephv1.CephClient{}
+	r.cephClientCephFSNode.Name = fmt.Sprintf("%s-%s", "cephuser-cephfs-node", r.storageConsumer.Name)
+	r.cephClientCephFSNode.Namespace = r.namespace
```

Comment:
Is there a reason why the new clients are prefixed with `cephuser` while the old ones are prefixed with `cephclient`?